### PR TITLE
Remove annotation instrumentation vertx dependency

### PR DESCRIPTION
**Pull request by trask**
Just some cleanup, thought I had removed annotation dependency from all the tests, but missed this

### CODE REVIEW

File: instrumentation/vertx-reactive-3.5/src/test/groovy/VertxReactiveWebServer.java [hunk 1]
```diff
@@ -98,26 +103,34 @@ public void start(final io.vertx.core.Future<Void> startFuture) {
...
   }
 
-  @WithSpan
   private void handleListProducts(final RoutingContext routingContext) {
-    HttpServerResponse response = routingContext.response();
-    Single<JsonArray> jsonArraySingle = listProducts();
-
-    jsonArraySingle.subscribe(
-        arr -> response.putHeader("content-type", "application/json").end(arr.encode()));
+    Span span = tracer.spanBuilder("handleListProducts").startSpan();
```
Comment (anuraaga) [inline instrumentation/vertx-reactive-3.5/src/test/groovy/VertxReactiveWebServer.java:107]: Can we put this change in another PR? Is it related to annotations?

Comment (anuraaga) [reply instrumentation/vertx-reactive-3.5/src/test/groovy/VertxReactiveWebServer.java:107]: Oops missed that

**Comment by trask** [comment]
oh, this is going to fix the current build failure in #929
